components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/internal/OAuth2ServiceComponentHolder.java

```diff
@@ -529,6 +532,25 @@ public void setRevocationProcessor(OAuth2RevocationProcessor revocationProcessor
         this.revocationProcessor = revocationProcessor;
     }
 
+    /**
+     * Get Indirect Token Revocation Handler.
+     *
+     * @return Indirect Token Revocation Handler
+     */
+    public IndirectTokenRevocationHandler getIndirectTokenRevocationHandler() {
+        if (indirectTokenRevocationHandler == null) {
```

Comment:
Add a new line after the method definition. Fix other places as well.

Reply:
Fixed with https://github.com/wso2-extensions/identity-inbound-auth-oauth/commit/69f8c55fb76f15cbfe22b3c551b17491dc1796b0